### ISSUE TITLE
CLI: echo the post multi-worker multiplication effective settings

### ISSUE THREAD

**Issue by oschaaf** [envoyproxy/nighthawk#84]

**Comment by oschaaf** [comment]
(Some context, this helps to avoid confusion, as some of the other OSS tools do not work like this; they try to distribute the configured rps/connections across the configured amount of threads/workers. But that delegates responsibility for aligning this configuration to the user, which seems error-prone. (e.g. one could then go on to configure a single connection for multiple threads)

**Comment by oschaaf** [comment]
This was resolved  in 52ab68de64b8ae841ef212a7839cde10d2ec6343 by ensuring we log the global targets in a CLI run, e.g:

```bash
./nighthawk_client --concurrency 2 --rps 5  127.0.0.1
[22:03:03.904036][10123][I] [source/client/process_impl.cc:180] Starting 2 threads / event loops. Test duration: 5 seconds.
[22:03:03.904082][10123][I] [source/client/process_impl.cc:182] Global targets: 200 connections and 10 calls per second.
[22:03:03.904089][10123][I] [source/client/process_impl.cc:186]    (Per-worker targets: 100 connections and 5 calls per second)
....
```

I still think we can summarize better to the end-user in the CLI's default ouput mode, especially now that we also capture the effective settings in our structured output, but that is more of a general enhancement round for the CLI output formatter.
